private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -182,8 +182,8 @@ class Dashboard extends React.Component<Props> {
                   <TemplateList
                     onClick={this.selectTemplate}
                     templates={recentlyViewedTemplates}
-                    displayComponents={{author: true, status: true, dateModified: true, templateName: true, version: false}}
-                  ></TemplateList>
```

Comment:
was this change necessary? or it is just fixing the styling?

Reply:
just styling